<h1>UK develops genetic early warning system for future pandemics</h1>
<div><strong>Date :</strong> 2023-04-02T07:00:16Z &nbsp; | &nbsp; <strong>Auteur :</strong> Robin McKie &nbsp; | &nbsp; <strong>Journal :</strong> World news</div>
<p>British researchers are developing a groundbreaking technology to monitor genetic changes in respiratory viruses as they circulate round the world. The system is to be used to pinpoint dangerous new variants as they emerge and act as an early warning system for new diseases and future pandemics.</p> <p>The team, which is based at the Wellcome Sanger Institute in Cambridgeshire, intends to make the technology cheap, easy to use and capable of being scaled up to provide global surveillance of a wide range of viruses. Targets would include influenza viruses, respiratory syncytial virus (RSV), coronaviruses and previously unknown pathogens.</p> <p>The ultimate aim of the project – <a href="https://www.sanger.ac.uk/group/respiratory-virus-and-microbiome-initiative/" title="">the Respiratory Virus and Microbiome Initiative</a> – is to create a system that would deploy DNA sequencing technology to identify all viral, bacterial and fungal species in a single sample collected from a nose swab from a patient.</p> <p>“Britain was at the leading edge of the genomic surveillance of Covid-19 and was responsible for about 20% of all the Sars-CoV-2 genomes that were sequenced across the planet during the pandemic,” said Ewan Harrison, who is leading the project at the Sanger Institute, a world-leading centre for genetics research and DNA sequencing.</p> <p>“The knowledge and data we generated allowed us to track – with unprecedented speed and accuracy – Sars-CoV-2, the virus responsible for Covid-19, and to monitor how it was changing. It was a wonderful aid in helping to fight the disease. Now we are aiming to contribute building a global genomic surveillance for all respiratory viruses. These, after all, are the agents most likely to trigger new pandemics,” Harrison added.</p> <p>An illustration of the threat of future pandemics is provided by coronaviruses. Three times in the past 20 years, a previously unknown coronavirus emerged to infect humans: <a href="https://www.theguardian.com/world/2023/mar/14/doctor-who-exposed-chinas-cover-up-of-sars-crisis-dies-aged-91" title="">Sars in China</a> and neighbouring countries; <a href="https://www.theguardian.com/technology/2013/sep/15/mers-next-global-pandemic" title="">Mers in the Middle East</a>; and <a href="https://www.theguardian.com/world/coronavirus-outbreak" title="">Covid-19, which affected the whole planet</a>.</p> <p>However, it was the use of genomic surveys during the Covid pandemic that revealed the technology’s remarkable potential. In December 2020, when there was a sudden rise in Covid cases in south-east England, the technology showed that this surge had been triggered by the appearance of a new, more infectious variant. Known originally as the Kent strain, it was later relabelled the Sars-CoV-2 Alpha variant.</p>    <p>“The discovery was a gamechanger,” said John Sillitoe, leader of the Sanger Institute’s genomic surveillance unit. “We generated genomic data very quickly and could see that this variant was transmitting at a very high rate. Suddenly, the world could see what genomics could do. It allows you to see changes in viruses much, much more quickly than by other methods, and now we are going to exploit that power.”</p> <p>The Sanger team is collaborating with the UK Health Security Agency, British academics, and other public health bodies on the project, with the aim of developing techniques that will allow them to sequence – from a single sample – not just one virus variant but any that might be infecting a patient. Typically, samples would be taken from individuals newly admitted to a hospital, where signs of a new, emerging disease are likely to first appear.</p> <p>However, such technology would have to be adaptable to laboratories around the world. “It would be no good if the UK and one or two other developed countries learned how to sequence respiratory virus genomes and no one else,” added Sillitoe. “If we don’t have this kind of surveillance globally, we’re not going to spot a dangerous new variant until it has already spread over much of the planet.”</p> <p>For the system to work across the globe, it would have to be the easiest, cheapest, fastest and most amenable to scaling that could be designed, said Sillitoe. “In many labs around the world, people have smaller sequencing machines and cannot sequence as many samples as we can at the Sanger. So we want the system to work equally well on those machines as it does here on our large volume devices.”</p> <p>Harrison told the <em>Observer</em> that three different versions of the technology were now being tested. “Over the coming months, we will be seeing if we can swap bits and possibly come up with a hybrid in the end. The aim would be to get a usable system in operation in a year, although there will always be room for improvements after that,” he added.</p> <p>“The crucial point is that we have got to develop a system that uses inexpensive reagents, does not require teams of highly-trained technicians and can be used at scale. Then we can really make a difference.</p> <p>“At present, for many respiratory viruses, we have sequenced maybe 1,000-2,000 of their genomes. We want to create hundreds of thousands of genomes for each of them in the near future. The information they will provide will be priceless, not just in tracking a new disease but in speeding up vaccine and drug development.”</p> <p>The team at the Sanger Institute is not the only group of scientists orking on expanding genomic surveys to cover many other emerging viruses. Centres in the US and in Germany are also working on similar projects.</p>